docs/dl-sandbox.md

```diff
@@ -8,54 +8,39 @@ Sandbox is for playing around with Scalar DL to roughly understand what Scalar D
-The key pair is used for communicating with Scalar DL network.
+## Download the example code
 
-Please note that we give you a generated key pair for ease of use in the Sandbox, but it is usually required to create your private key in your own environment.
+Download the example code from our Github repository (the following command will clone the Scalar DL client SDK).
 
-## Before running your first contract 
-
-Let's clone Scalar DL client SDK to interact with Scalar DL network.
 ```
 $ git clone https://github.com/scalar-labs/scalardl-client-sdk.git 
 ```
-You can put your downloaded zip in the directory and unzip it.
 
-Scalar DL manages data as a set of assets, where each asset is composed of a history of data identified by a key called `asset_id` and historical version number called `age`.
-`asset_id` is an arbitrary, but unique, string specified by users to manage their assets. However, the Sandbox is a shared environment that anyone can access,
-so please be careful about choosing an appropriate name so that it will not conflict with `asset_id`s chosen by other users.
-One recommended way to do this is to append your username to your asset name as in `<username>_your-asset-name`.
+Then copy your `client.properties` file, certificate, and private key to the `conf` directory in the repository.
 
-## Run your first contract
+## Register your certificate
+
+Next, let's register your certificate in the Scalar DL network. The registered certificate will allow you to register and execute contracts, and will also be used for tamper detection of the data stored in the network.
 
-First, you need to configure some properties to interact with Sandbox.
-It is recommended to use the properties file which can be downloaded from the site.
-If you want to configure by yourself, pleae update the following properties in addition to the required properites to interact with Sandbox.
-Then you are ready to follow [the doc](dl-getting-started.md) to run your first contract.
-(Again, please note that you need to choose non-conflicting asset ids to properly use the Sandbox.)
+You may register your certificate with the following command.
 
 ```
-# A host name of Scalar DL Sandbox network
-scalar.ledger.client.server_host=sandbox.scalar-labs.com
+$ client/bin/register-cert -properties conf/client.properties
+```
 
-# A port number of Scalar DL Sandbox network
-scalar.ledger.client.server_port=443
 
-# A flag to enable TLS communication.
-scalar.ledger.client.tls.enabled=true
+## Run your first contract
 
-# An authorization credential. (e.g. authorization: Bearer token)
-# If this is given, clients will add "authorization: <credential>" http/2 header.
-scalar.ledger.client.authorization.credential=Bearer <your-token>
-```
+Now you are ready to follow the "[Run your first contract](dl-run-first-contract.md)" document to run your first contract.
+
+
+##### Note on choosing asset ids in the Sandbox
```

Comment:
I think this should be noted before going to `Run your first contract`, otherwise the user will just copy and get some inconsistent results since the asset id might have been created and updated.
Also, I think we need a note on choosing contract id.